src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The href attribute in the link tag should point to a CSS file, not a SCSS file. SCSS files need to be compiled into CSS before they can be used in an HTML document. Please ensure that the href attribute points to the correct CSS file.